automation/terraform/modules/testnet-alerts/templates/testnet-alert-rules.yml.tpl

```diff
@@ -129,7 +129,7 @@ groups:
       runbook: "https://www.notion.so/minaprotocol/Nodes-out-of-sync-0f29c739e47c42e4adabe62a2a0316bd"
 
   - alert: LowPeerCount
-    expr: min by (testnet) (Coda_Network_peers ${rule_filter}) < 3
+    expr: quantile by (testnet) (0.5, Coda_Network_peers ${rule_filter}) < 3
```

Comment:
I make it quantile instead of min because when we take the min, the collected data seems to be affected by some abnormal nodes. Looking at the graph: https://o1testnet.grafana.net/d/4BrFOXyMk/alerts?orgId=1&refresh=5s&var-testnet=berkeley&from=now-2d&to=now&viewPanel=16. When the alert gets fired, the curve usually jumped from 20-ish to 0 all of a sudden.